client/app/vrack/add/vrack-add.controller.js

```diff
@@ -43,10 +45,24 @@ angular.module("managerApp").controller("VrackAddCtrl",
 
         function init () {
             self.loaders.loading = true;
-            self.getVrackContract()
-            .finally(function () {
-                self.loaders.loading = false;
-            });
+            self.vrackOrderUrl = null;
+
+            var promise = {
```

Comment:
`var`? 😨

Reply:
A perfect timing would be when we switch to ESLint Airbnb base config. Isn't it?